src/about/settings/index.js

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define((require, exports, module) => {
+
+  'use strict';
+
+  const Component = require('omniscient');
+  const {render} = require('browser/core');
+  const {DOM} = require('react');
+  const {fromJS} = require('immutable');
+
+  const Settings = Component('Settings', state => {
+
+    let rows = [];
+    for (let [name,value] of state) {
+      rows.push(DOM.div({
+        key: name,
+        className: 'row'
+      }, [
+        DOM.span({
+          key: 'name.' + name,
+          className: 'name',
+          title: name
+        }, name),
+        DOM.span({
+          key: 'value.' + name,
+          className: 'value',
+          title: value
+        }, value)
+      ]));
+    }
+
+    return DOM.div({
+      key: 'table',
+      className: 'table'
+    }, rows);
+
+  });
+
+  let state;
+
+  const update = () => render(Settings, state, document.body);
```

Comment:
Idea was that one would call `render` to spawn up a render loop, this works but seems little odd. I would advice doing something along these lines instead:

```js
const settings = render(Settings, state, document.body)
```

Maybe `settings` isn't a great name, I'll let you come up with a good one. Other one that comes to my mind is `app`.